utils/README.md

```diff
@@ -35,7 +35,7 @@ precision= true positives /
 recall= true positives /
         true positives + false negatives
 ```
-
+<!--
```

Comment:
Why?

Reply:
I noticed lots of weird behaviors with other metrics specifically with multiclass evaluation. Only dice and IoU have been tested extensively, so I decided to exclude them for now.